configuration/rules-dsl.md

```diff
@@ -92,6 +96,8 @@ then
 end
 ```
 
+Note that the rule name must match the filename of the rule, minus the extension ".rules"
```

Comment:
Could it be that you are confusing rules files with sitemap files?
The sentence is not correct, rules files can be named however you want.

Reply:
Oh boy.  This is really confusing.  Yep, there is an error here in the line I added - it should NOT say that the name has to match the rule.  But again, I do not know how I made these changes, so it is difficult for me to go back and fix it.  can you do it please @ThomDietrich ?